app/models/govbox/message.rb

```diff
@@ -140,6 +145,15 @@ def self.add_upvs_related_tags(message, govbox_message)
     add_delivery_notification_tag(message) if message.can_be_authorized?
   end
 
+  def self.migrate_tags_from_draft(message, message_draft)
```

Comment:
migrate znie divne, skor `copy_tags_from_message` (asi je jedno ci to je draft alebo nie nie?

Reply:
premenovala som